libs/SalesforceSDK/src/com/salesforce/androidsdk/ui/LoginActivity.java

```diff
@@ -422,6 +425,21 @@ public void finish(UserAccount userAccount) {
             intent.putExtra(IDPAccountPickerActivity.USER_ACCOUNT_KEY, userAccount.toBundle());
             sendBroadcast(intent);
         }
+
+        // If the IDP app specified a component to launch after login, launches that component.
+        if (!TextUtils.isEmpty(spActivityName)) {
```

Comment:
Launches the component that was passed in from the IDP at the end of the login flow.